evil-dataset/README.md

```diff
@@ -0,0 +1,78 @@
+# Evil Dataset
+
+This directory is intended to house the "evil dataset"--a collection of data & associated queries that test edge cases and behavior changes between versions.
+
+## Datapoint Library
+The datapoints are contained in `/datapoint-library`. The directory name is the unique identifier for the test case.
+
+Within the datapoint-library directory, the directory structure should be as follows:
+
+```
+datapoint-library/
+├─ example-datapoint/
```

Comment:
[blocker] Happy to iterate on the design as appropriate, but I'm curious to hear your thoughts on the number of "datapoints" you think we'll have in the medium and long term and whether we'll want to stick with this approach of separate directory structures for each of them?  Or are we thinking of making a custom file format that encompasses all the data for a "datapoint"?  Or maybe multiple "datapoints"?